args.js

```diff
@@ -14,7 +14,7 @@ const optionsBook = {
         acceptedValues: range(1, 10000)
     },
     logpath: {
-        defaultValue: './activity.log',
+        defaultValue: `${__dirname}/activity.log`,
```

Comment:
We'll need to normalize all of our path names throughout the app with the [`path`](https://nodejs.org/api/path.html) module. The `path` module will create operating system aware paths (windows paths are different than mac and linux paths. for one, they use `\` instead of `/`).